teal/packer/display.tl

```diff
@@ -564,362 +469,129 @@ display.task_start = vim.schedule_wrap(function(self: Display, plugin: string, m
   if not valid_display(self) then
     return
   end
-  if self.marks[plugin] then
-    self:task_update(plugin, message)
-    return
-  end
-  display.running = true
-  set_lines(self, HEADER_LINES, HEADER_LINES, {
-    fmt(' %s %s: %s', config.display.working_sym, plugin, message),
-  })
 
-  self.marks[plugin] = self.marks[plugin] or {}
-  self.marks[plugin].start = set_extmark(self.buf, self.ns, nil, HEADER_LINES, 0)
+  local item = self.items[plugin]
+  item.status = 'running'
+  item.message = message
+
+  render_task(self, plugin)
 end)
 
 --- Decrement the count of active operations in the headline
-local decrement_headline_count = vim.schedule_wrap(function(disp: Display)
+local function decrement_headline_count(disp: Display)
```

Comment:
I'm probably missing some change with how this is called, but: why doesn't this need `schedule_wrap` any longer?

Reply:
Ah - because you've verified that it's only called from within already-wrapped functions.